src/workloads/docs/Generators.yml

```diff
@@ -102,6 +107,12 @@ Actors:
             # options to narrow the range of IP addresses. Every IP address is equally likely.
             ip: {^IP: {}}
 
+            # Generate the thread ID as a numbers. This will be constant for a given actor, but
+            # unique across actors
+            threadNum: {^ThreadID: {}}
```

Comment:
this'll be the new ActorId syntax, ya?

Reply:
Yeah. Thanks!

Reply:
And it made the CI break, so that's nice that it would have been caught. Fixed now.